test/livebook/storage/ets_test.exs

```diff
@@ -3,6 +3,16 @@ defmodule Livebook.Storage.EtsTest do
 
   alias Livebook.Storage.Ets
 
+  setup_all do
+    # clear all possible stale test's entities
+    [:insert, :fetch_key, :fetch, :delete, :all, :persistence]
+    |> Enum.each(fn namespace ->
+      namespace
+      |> Ets.all()
+      |> Enum.each(&Ets.delete(namespace, &1.id))
+    end)
+  end
+
```

Comment:
Since we remove the storage file we no longer need to clear right?

```suggestion
```

Reply:
Right, forgot about it :)